test/webrtc_test.rb

```diff
@@ -31,7 +31,7 @@
         @task2.properties.signaling_server_name = "127.0.0.1:3012"
     end
 
-    it "configure and starts the tasks" do
+    it "configure, starts and send message between the tasks" do
```

Comment:
Don't attempt to cram every test in a single `it`

Instead, test one thing after the other in different `it` calls, optionally using helpers to avoid too much repetition

Tests that I see

```
it "emits start only after the data channel is fully established" # check the status port
it "transmits data from the active to the passive task once started"
it "transmits data from the passive to the active task once started"
it "detects that the channel has been closed from the passive side" # task2 should get into exception if task1 is stopped
it "detects that the channel has been closed from the active side" # task1 should get into exception if task1 is stopped
it "successfully reestablishes connection after a stop/start cycle" # to make sure the component is functional if we stop and then start it again
```

Reply:
Do you really think that we should go into an exception event if the datachannel is closed? Because the component is not going to that kind of event in that scenario.

Reply:
Should we put an exception to represent that on the callback?